### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute '__dict__'

### ISSUE THREAD

**Issue by sobolevn** [lensvol/astboom#2]
Version: 0.2.2

Reproduction:

```
» echo "keys={'a': 1}; {**keys}" > ex.py

» astboom < ex.py
Failed to read source from command line, trying to read it from STDIN:
========================================================================
======================================================================== 

Traceback (most recent call last):
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/bin/astboom", line 10, in <module>
    sys.exit(cli())
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/main.py", line 60, in cli
    source_tree = engine.process(source)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/visualizers/ast.py", line 63, in process
    return {class_name(module): self._traverse(module)}
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/visualizers/ast.py", line 37, in _traverse
    traversed_items = {
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/visualizers/ast.py", line 38, in <dictcomp>
    f"[{i}] {class_name(item)}": self._traverse(item)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/visualizers/ast.py", line 34, in _traverse
    (f"{attr}: {class_name(value)}", self._traverse(value))
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/visualizers/ast.py", line 37, in _traverse
    traversed_items = {
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/visualizers/ast.py", line 38, in <dictcomp>
    f"[{i}] {class_name(item)}": self._traverse(item)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.8/site-packages/astboom/visualizers/ast.py", line 25, in _traverse
    for attr, value in sorted(node.__dict__.items(), key=lambda p: p[0]):
AttributeError: 'NoneType' object has no attribute '__dict__'

```

**Comment by lensvol** [comment]
Thank you! Will work on it soon.

**Comment by lensvol** [comment]
Fixed in commit [86b57f9](https://github.com/lensvol/astboom/commit/86b57f9506df1a57e4740f25102b685d0a980ca6), released in [0.4.2](https://pypi.org/project/astboom/0.4.2/).

**Comment by sobolevn** [comment]
@lensvol dependabot is not working right now. Do you want to send a PR with version update to https://github.com/wemake-services/wemake-python-styleguide ?
